tests/python_package_test/test_dual.py

```diff
@@ -26,9 +27,10 @@ def test_cpu_and_gpu_work():
 
     params_gpu = params_cpu.copy()
     params_gpu["device"] = "gpu"
-    params_gpu["gpu_use_dp"] = True
+    params_gpu["gpu_use_dp"] = (platform.machine() == "x86_64")
```

Comment:
Could you please explain why we don't set `gpu_use_dp=True` on aarch64?

Reply:
The reason is that double-precision floats are not supported with PoCL on aarch64.

Reply:
Ah, I see! Thanks for the explanation.

Could you please add comment "double-precision floats are not supported with PoCL on aarch64" in the code to make it clear for everyone reading the code?

Reply:
Done!